omod/src/main/resources/org/openmrs/module/fhir2/narratives/AllergyIntolerance.html

```diff
@@ -16,11 +16,11 @@
     <table class="hapiPropertyTable" th:if="${not resource.empty}">
         <tbody>
         <tr th:if="${resource.id}">
-            <td>Id:</td>
+            <td th:text="#{narrative.propertyName.id} ?: 'Id'"></td>
```

Comment:
Using a default value if the message is absent, so that populating the `messages.properties` won't be required for English language (and the default would be English too).

Reply:
I'd make sure that these defaults also exist at least in the base `message.properties` file.

Reply:
So should I rather remove the defaults from the templates or keep them at both the places?